android/build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlinVersion = '1.6.0'
+    ext.kotlinVersion = '1.7.10'
```

Comment:
I remember we downgrade this version for some problem we had recently... @Oscar-Pepper do you remember this?

Reply:
Let me try this version... maybe with some new other lib version could work...

Reply:
I have no problem with Android Studio and the new kotlin version.